convert2rhel/actions/system_checks/duplicate_packages.py

```diff
@@ -31,10 +32,17 @@ def run(self):
         super(DuplicatePackages, self).run()
 
         logger.task("Prepare: Check if there are any duplicate installed packages on the system")
-        output, _ = utils.run_subprocess(["/usr/bin/package-cleanup", "--dupes", "--quiet"], print_output=False)
+        output, ret_code = utils.run_subprocess(["/usr/bin/package-cleanup", "--dupes", "--quiet"], print_output=False)
         if not output:
             return
-
+        # For el7 machines we have to depend on this output to know the check failed
+        if system_info.version.major == 7 and "[Errno -2] Name or service not known" in output:
```

Comment:
can't we check only for `errno -2` or `name or service not known` in the output?

Also, let's normalize the output to `.lower()` to avoid case-sensitive problems